jaffle_shop/models/final/finance/fnl_returned_order_value.sql

```diff
@@ -0,0 +1,18 @@
+WITH returned_orders AS (
+  SELECT * 
+  FROM {{ ref('wh_orders') }}
```

Comment:
These two CTE's are breaking a SQL/gatekeeper rule - can you figure out which one it is? :)